float64 theta
float64 theta_dot
float64 x_position
float64 x_dot
